week-1-practise/level-1/9-function-output-2.js

```diff
@@ -11,4 +11,11 @@ console.log(result);
 // - What is the function name?
 // - Where is the function called? with what parameters?
 // - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - How can this function be improved?
+/*-Response
+the programme concatenate two parameter
+the function take two parameter
+ the name is concatenate
+ the function is called in line 5, code and your
+
+the function return codeyour*/
```

Comment:
After you amended the function, did you try running it to see whether it did as you expected it to? You might want to do that and keep a close eye on any errors that appear in the console.

Also, you don’t appear to have answered the final question on improving the function - it’s looking for a narrative answer, not just changes to the code itself.